include/superior_mysqlpp/prepared_statements/prepared_statement_base.hpp

```diff
@@ -68,23 +94,39 @@ namespace SuperiorMySqlpp
             }
 
         protected:
+            /**
+             * This method implements the store() specialized behavior.
+             */
             void storeOrUse()
             {
                 this->statement.storeResult();
             }
         };
 
+        /**
+         * Specialization of StatementBase where statement results are "used" - not
+         * fetched all at one.
```

Comment:
The code design, how prepared statement distinguishes between *store* and *use* results should be explained somewhere.

Reply:
I hope description of tparam for all related classes and in general usage description will be enough.